docs/predictive-analytics/model-retraining/process.sql

```diff
@@ -0,0 +1,66 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create stream of predictions
+CREATE STREAM PREDICTED_WEIGHT(
+  "Fish_Id" VARCHAR KEY,
+  "Species" VARCHAR,
+  "Height" DOUBLE,
+  "Length" DOUBLE,
+  "Prediction" DOUBLE
+  )
+WITH(
+  KAFKA_TOPIC = 'kt.mdb.weight-prediction', 
+  VALUE_FORMAT = 'JSON',
+  PARTITIONS = 6
+);
+
+-- Create stream of actual weights
+CREATE STREAM ACTUAL_WEIGHT(
+  "Fish_Id" VARCHAR KEY,
+  "Species" VARCHAR,
+  "Weight" DOUBLE
+  )
+WITH(
+  KAFKA_TOPIC = 'kt.mdb.machine-weight', 
+  VALUE_FORMAT = 'JSON',
+  PARTITIONS = 6
+);
+
+-- Create stream joining predictions with actual weights
+CREATE STREAM DIFF_WEIGHT
+WITH(
+  KAFKA_TOPIC = 'diff_weight', 
+  VALUE_FORMAT = 'JSON'
+)
+AS SELECT
+   'Key' AS "Key",
+   PREDICTED_WEIGHT."Fish_Id" AS "Fish_Id",
```

Comment:
nit (as above): `predicted_weight."Fish_Id"` ?

Same in next lines and other places (won't add more comments)

Reply:
Done